### ISSUE TITLE
proxyRes not working

### ISSUE THREAD

**Issue by zwhitchcox** [BrowserSync/browser-sync#1073]
This is my relevant code:

```js
import browserSync   from 'browser-sync'
const bs = browserSync.create()

let server;
bs.init({
  proxy: {
    target: 'localhost:8000',
    proxyRes: [
      (res,req,next) => {
        req.body += 'hello'
        next()
      }
    ]
  },
})
```

it says next is not a function

Also, it takes forever browsersync to start. There's only two files to watch, although I don't even really want browsersync to watch any files, but it takes like 10 - 15 seconds to start

**Comment by zwhitchcox** [comment]
Also, snippet options doesn't work either. It literally just didn't do anything

**Comment by zwhitchcox** [comment]
Idk what I did, but it's working now

Code for anyone needing a fix:

```js
import browserSync   from 'browser-sync'
const bs = browserSync.create()
bs.init({
  proxy: {
    target: 'localhost:8000',
  },
  snippetOptions: {
    rule: {
      match: /.*/i,
      fn: function (snippet, match) {
        return snippet + match
      }
    }
  }
})
```

**Comment by zwhitchcox** [comment]
Nevermind, not working. I forgot I added a `body` tag to the response.

Ok, I think snippet options only modifies where the snippet is. Basically, what I want to do is add the snippet, whether there is a body tag or not. It looks like what I need to use is proxyRes, which does not work, even though, I basically have the same code as the [documentation](https://www.browsersync.io/docs/options/).

So, looks like I'm back to my original problem.

**Comment by zwhitchcox** [comment]
So looks like there is no way to modify the response, which renders browserify completely useless for this project, unless I'm missing something

**Comment by alliv8** [comment]
@zwhitchcox how did you finally solve this? I'm looking for a similar solution, where I want to modify the served html and css files to remove `-2x || @2x`.

I was able to remove it from the HTML using 
`rewriteRules: [
      {
        match: /-2x\.|@2x\./g,
        replace: "."
      }
    ]`

But haven't been able to modify the css response yet. I checked  [https://github.com/expressjs/serve-static](serveStatic) to see if I could add a middleware that would let me do it but couldn't.

Thinking I'll have to do this by adding nginx before browsersync instead ~
